### PR TITLE
Fix pypi

### PR DESCRIPTION
**Pull request by gbarter**
## Description and Purpose
This should fix and restore a full pip/pypi workflow.

## Type of change
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [x] Maintenance update
- [ ] Other (please describe)

TODO Items General:
- [ ] Add example/test for new feature
- [ ] Update registry
- [x] Run testing

TODO Items API Change:
- [ ] Update docs with API change
- [ ] Run update_rosco_discons.py in Test_Cases/
- [ ] Update DISCON schema

### CODE REVIEW

**Comment by gbarter** [comment]
@dzalkind - Here are the steps I would suggest:

1. Staging these changes into `develop` first
2. Then we should bring in Marc's #338 
3. Then PR from `develop` to `main`
4. Then tag the 2.9.2 release (already set the `pyproject.toml` to that value)

I'll defer to you, but if it helps, I can push these steps through .

File: .github/tools/cibw_repair_wheel_command_windows.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -e
```
Comment (dzalkind) [inline .github/tools/cibw_repair_wheel_command_windows.sh:3]: Out of curiosity, what's happening here?

**Review by dzalkind — APPROVED** [review]
Thank you for helping with this!

File: .github/tools/cibw_repair_wheel_command_windows.sh (hunk 1 above)
Comment (gbarter) [reply .github/tools/cibw_repair_wheel_command_windows.sh:3]: I borrowed that file from another Github project that had a very similar package setup.  I cannot take credit for anything there.  For those two lines you highlighted:

https://askubuntu.com/questions/1402718/is-the-same-usr-bin-env-bash-than-bin-bash

https://linuxcommand.org/lc3_man_pages/seth.html

Comment (dzalkind) [reply .github/tools/cibw_repair_wheel_command_windows.sh:3]: Ah, okay.  I meant the whole file, but now I see that those lines are quite highlighted from this view.

Comment (gbarter) [reply .github/tools/cibw_repair_wheel_command_windows.sh:3]: Oh, sorry.  There is some problems with bundling a compiled library in a python wheel in windows and this file is supposed to bundle it outside of `pip install`

**Pull request merged**